packages/gatsby-core-utils/package.json

```diff
@@ -37,6 +37,7 @@
     "@babel/cli": "^7.7.5",
     "@babel/core": "^7.7.5",
     "babel-preset-gatsby-package": "^0.2.17",
-    "cross-env": "^5.2.1"
+    "cross-env": "^5.2.1",
+    "@types/ci-info": "2.0.0"
```

Comment:
@danielkov You shouldn't manually add dependencies.
They normally appear in alphabetical order

Reply:
updated